Evaluation/Input/evaluation_plan.json

```diff
@@ -0,0 +1,579 @@
+{
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json",
+    "Projects": [
+        {
+            "Id": "Fluvoxamine-Model",
+            "Path": "https://raw.githubusercontent.com/LMFuhr/Fluvoxamine-Model/master/Fluvoxamine-Model.json",
```

Comment:
Ref to OSP-external repo. Not good.
As long as there is no release of the model it can be replaced by `../Fluvoxamine-Model.json`